test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -51,6 +52,12 @@ public void elementsAreUnique() throws Exception {
         // all objects unique
         assertAreUnique("abc", "ab", "a");
         assertAreUnique(1, 2);
+        assertAreUnique('a', "a");
+        
+        //two lists
+        List<Object> newList = Arrays.asList("a");
```

Comment:
What we wanted you to do was to write a test method such as elementsAreUnique(){...}

Note the naming convention for test methods (the first part is feature/method under test), second is the testing scenario and the third is the expected behaviour. 
E.g. isAnyNull_emptyList_falseReturned
Part 2 and 3 of the test name can be removed to indicate an increased test scope